R/upkeep.R

```diff
@@ -243,6 +243,34 @@ tidy_upkeep_checklist <- function(last_upkeep = last_upkeep_year(),
     )
   }
 
+  if (last_upkeep <= 2025) {
+    bullets <- c(
+      bullets,
+      "### 2025",
+      "",
+      todo("`usethis::use_air()` <https://posit-dev.github.io/air/>"),
+      todo('`usethis::use_package("R", "Depends", "4.1")`'),
+      todo("Switch to the base pipe (`|>`)"),
```

Comment:
Do we need to add `(or create an issue)` or is that now implied?

Reply:
I feel like it's implied at this point.